src/main/java/teammates/ui/webapi/GetOngoingSessionsAction.java

```diff
@@ -28,19 +27,27 @@ public JsonResult execute() {
         long startTime;
         try {
             startTime = Long.parseLong(startTimeString);
-            //test for bounds
+        } catch (NumberFormatException e) {
+            throw new InvalidHttpParameterException("Invalid startTime parameter", e);
+        }
+        try {
+            // test for bounds
             Instant.ofEpochMilli(startTime).minus(Const.FEEDBACK_SESSIONS_SEARCH_WINDOW).toEpochMilli();
-        } catch (NumberFormatException | ArithmeticException e) {
+        } catch (ArithmeticException e) {
             throw new InvalidHttpParameterException("Invalid startTime parameter", e);
         }
```

Comment:
We can't combine them because the second parameter is of different class. Java compiler is not smart enough to know that there are two different constructors for the two different exceptions being caught here; it will be treated as `RuntimeException` which is the lowest common denominator, and we don't want such generic exception to be catchable by this particular custom exception. At times like this I wish something like Typescript's union type exists, but not like it's a big deal anyway.

Reply:
Ahh, I see!